### PR TITLE
Adding release notes for 2.8.0

### PR DESCRIPTION
**Pull request by poojaSprophecy**

### CODE REVIEW

File: docs/release_notes/feb2023.md [hunk 1]
```diff
@@ -8,6 +8,67 @@ tags:
...
+## 2.8.0.0 (February 28, 2023)
+
+- Prophecy Python libs version: 1.4.2
+- Prophecy Scala libs version: 7.0.7
+
+### Updates {#UpdatesRelease280}
+
+#### No team required on sign up
+
+Previously, A user was asked to enter a unique team when signing up for Prophecy. To improve the sign-up experience, requirement of providing a Team is now removed.
```
Comment (slyons) [inline docs/release_notes/feb2023.md:20]: "To improve the sign-up experience, the requirement of providing a Team is now removed" -> "To improve the sign-up experience the requirement has been removed"

File: docs/release_notes/feb2023.md [hunk 2]
```diff
@@ -8,6 +8,67 @@ tags:
...
+
+- Prophecy Python libs version: 1.4.2
+- Prophecy Scala libs version: 7.0.7
+
+### Updates {#UpdatesRelease280}
+
+#### No team required on sign up
+
+Previously, A user was asked to enter a unique team when signing up for Prophecy. To improve the sign-up experience, requirement of providing a Team is now removed.  
+Now the user can just sign up by giving his Name, Email and Company name. He can be added to a team by Admin post sign up.
```
Comment (slyons) [inline docs/release_notes/feb2023.md:21]: his -> their
he -> They
"Admin post sign up" -> "Admin after signing up"

File: docs/release_notes/feb2023.md [hunk 3]
```diff
@@ -8,6 +8,67 @@ tags:
...
+
+![New_Sign_up](img/new_sign_up.png)
+
+#### Sorting on tables
+
+All metadata tables will now have options to sort on the columns. By default, the entities in table will be sorted on Modified at or Created At time in decreasing order.
```
Comment (slyons) [inline docs/release_notes/feb2023.md:27]: "Modified at" -> `` `Modified at` ``
"Created At" -> `` `Created At` ``

File: docs/release_notes/feb2023.md [hunk 4]
```diff
@@ -8,6 +8,67 @@ tags:
...
+
+![Sorting](img/Sorting.gif)
+
+#### Use a Cluster in Current tab
+
+A User editing Prophecy Pipelines in multiple tabs, can now detach the cluster Attached to a Pipeline in another tab, and attach to current Pipeline by clicking the same cluster.
```
Comment (slyons) [inline docs/release_notes/feb2023.md:33]: "tabs, can" -> "tabs can"
"tab, and" -> "tab and"
"attach to current Pipeline" -> "attach the current Pipeline"

File: docs/release_notes/feb2023.md [hunk 5]
```diff
@@ -8,6 +8,67 @@ tags:
...
+He will see a warning as below before the cluster is detached in previous tab.
```
Comment (slyons) [inline docs/release_notes/feb2023.md:34]: He -> They
"will see a warning as below" -> "will see this warning"

**Review by slyons — CHANGES_REQUESTED** [review]

File: docs/release_notes/feb2023.md (hunk 2 above)
Comment (poojaSprophecy) [reply docs/release_notes/feb2023.md:21]: Oops, Should have kept in mind the gender-neutral pronouns. Thank you for pointing out.

File: docs/release_notes/feb2023.md [hunk 6]
```diff
@@ -17,41 +17,43 @@ tags:
...
 
-If there were running Jobs in another tab, he would be prompted to cancel the existing Jobs before switching.
+If there were running Jobs in another tab, they will be prompted to cancel the existing Jobs before switching.
 ![Attach-cluster-warning](img/Attach_cluster_job_warning.png)
 
 #### Link to Databricks Workflow in Released Prophecy Jobs
 
-In Job editor view, Added the link to the corresponding Databricks Workflow. Please note, this will only come if the Job is enabled and released.
+In Job editor view we've added the link to the corresponding Databricks Workflow.
+
```
Comment (JagmeetKaur) [inline docs/release_notes/feb2023.md:43]: [slyons](https://github.com/slyons) [yesterday](https://github.com/SimpleDataLabsInc/prophecy-docs/pull/104#discussion_r1122111704)

"In Job editor view, Added" -> "**In the Job editor we've added**"

**Review by JagmeetKaur — APPROVED** [review]
Please fix the minor review comment from yesterday.

Approving the PR

File: docs/release_notes/feb2023.md [hunk 7]
```diff
@@ -17,41 +17,43 @@ tags:
...
 #### Use a Cluster in Current tab
 
-A User editing Prophecy Pipelines in multiple tabs, can now detach the cluster Attached to a Pipeline in another tab, and attach to current Pipeline by clicking the same cluster.
-He will see a warning as below before the cluster is detached in previous tab.
+A User editing Prophecy Pipelines in multiple tabs can now detach the cluster Attached to a Pipeline in another tab and attach the current Pipeline by clicking the same cluster.
+They will see this warning before the cluster is detached in previous tab.
 ![Attach-cluster](img/Attach-cluster1.png)
 
-If there were running Jobs in another tab, he would be prompted to cancel the existing Jobs before switching.
+If there were running Jobs in another tab, they will be prompted to cancel the existing Jobs before switching.
```
[the remark this answers is not in the document]
Comment (atbida) [reply docs/release_notes/feb2023.md:37]: Prophecy is a web interface application, so users should interact using a single browser tab. We've clarified this user experience by prompting the user to select one browser tab. Only one browser tab can edit a Pipeline, attach to a Spark cluster, or run Jobs. Users will see warning messages before detaching a cluster in a previous tab, or to cancel existing Jobs before switching tabs.

File: docs/release_notes/feb2023.md [hunk 8]
```diff
@@ -8,6 +8,69 @@ tags:
...
+
+#### Allow Run on main branch
+
+Editing your Pipeline on Main branch is not allowed. Interactive run and Partial run on Gems is allowed when viewing a Pipeline on Main branch. Interims will also work as expected.
+
+![Run_main_branch](img/Run_Main_branch.png)
+
+#### Cluster restart after setting dependencies
+
+Improved the cluster restart behaviour when adding a new Pipeline dependency. When a new Pipeline dependency is added, cluster will be detached and prompt for a restart as shown below. A small subset of users were experiencing cluster restart loops when using certain dependencies and this has been addressed
```
Comment (atbida) [inline docs/release_notes/feb2023.md:70]: I suggest to remove this text: 
 "A small subset of users were experiencing cluster restart loops when using certain dependencies and this has been addressed"

Comment (slyons) [reply docs/release_notes/feb2023.md:70]: I suggested adding that, @atbida. What's your thoughts on the matter?

**Review by slyons — APPROVED** [review]

**Pull request merged**